libs/design-system/navigation/header/src/lib/menu-content/menu-content.component.html

```diff
@@ -61,29 +55,17 @@
 
 <ng-template hraMenuItem let-item #itemTemplate>
   @let url = resolveUrl(item.url, item.external);
-  @if (url.isAbsolute) {
-    <a
-      mat-button
-      hraSecondaryButton
-      class="item-label"
-      [attr.href]="url.value"
-      [attr.target]="item.target ?? ''"
-      rel="noopener noreferrer"
-      hraClickEvent
-      [hraFeature]="item.label | slugify"
-    >
-      {{ item.label }}
-    </a>
-  } @else {
-    <a
-      mat-button
-      hraSecondaryButton
-      class="item-label"
-      [routerLink]="url.value"
-      hraClickEvent
-      [hraFeature]="item.label | slugify"
-    >
-      {{ item.label }}
-    </a>
-  }
+  <a
+    mat-button
+    hraSecondaryButton
+    class="item-label"
+    [hraLink]="url.value"
```

Comment:
Use `group.url` & `group.external`

Reply:
Yes, I copied this from a previous comment and missed updating the variable name